client/lib/js/constants/api_urls.js

```diff
@@ -8,7 +8,8 @@ const ApiUrls = {
     'assets_for_account': API_PATH + 'accounts/%(accountId)s/assets/',
     'assets': API_PATH + 'assets/',
     'assets_detail': API_PATH + 'assets/%(assetId)s/',
-    'assets_transfer': API_PATH + 'assets/%(assetId)s/%(cid)s/transfer/'
```

Comment:
This is using the [sprintf-js](https://github.com/alexei/sprintf.js#named-arguments) notation. I do find the template strings notation more readable, but its use could also lead to confusion since they're getting used out of context with something like `ApiUrls`.